open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -25,7 +25,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<p>and aggregation function <code>sum()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>DONE</p>